src/Lunr/Vortex/FCM/FCMApnsPayload.php

```diff
@@ -20,7 +17,7 @@
  *     image?: string
  * }
  * @phpstan-type FcmApnsConfig array{
- *     headers?: array<string,string>,
+ *     headers?: array<string,string|int>,
```

Comment:
Not according to the upstream documentation https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#ApnsConfig

`headers` is `map (key: string, value: string)`

The example has numbers also as a string there